distribution/lib/Standard/Base/0.0.0-dev/src/Data/Text/Matching.enso

```diff
@@ -1,27 +1,29 @@
-from Standard.Base import all
+import project.Data.Numbers.Integer
+import project.Data.Pair.Pair
+import project.Data.Range.Extensions
+import project.Data.Text.Text
+import project.Data.Vector.Vector
+import project.Error.Problem_Behavior.Problem_Behavior
+import project.Panic.Panic
+import project.Panic.Wrapped_Dataflow_Error
 
-from Standard.Base.Error.Problem_Behavior import Report_Warning
-from Standard.Base.Error.Common import Wrapped_Dataflow_Error_Data
-
-# TODO Dubious constructor export
-from project.Data.Text.Matching.No_Matches_Found import all
-from project.Data.Text.Matching.No_Matches_Found export all
+from project.Data.Boolean import Boolean, True, False
```

Comment:
Makes me think if we should have a `Standard.Core` covering the most primitive types, essentially replacing the `micro-distribution`.

Then in proper `Standard.Base` we could do `from Standard.Core import all` instead. And ofc we'd re-export `Core` in `Base`, so that regular users would just use `Base`.